src/blocks/NoRecentDataWarning.svelte

```diff
@@ -0,0 +1,43 @@
+<script>
+  import { formatDateYearDayOfWeekAbbr } from '../formats';
+  /**
+   * @type {import("../stores/params").DateParam}
+   */
+  export let date;
+  /**
+   * @type {import("../stores/params").DateParam}
+   */
+  export let minMaxDate;
+  /**
+   * @type {import('../stores/params').SensorParam}
+   */
+  export let casesSensor;
+  /**
+   * @type {import('../stores/params').SensorParam}
+   */
+  export let deathSensor;
+  /**
+   * @type {import('../stores/params').SensorParam}
+   */
+  export let hospitalAdmissionSensor;
+
+  let todaysDate = new Date();
```

Comment:
do we need `todaysDate`?  can we use `date` instead?  i think theyre set to the same value, right?

Reply:
It is a tricky way as `date` is stored in some kind of "storage" and it is global, so when date value is changed from code/web page/url, it is reflected in that variable's value.
So I had to create one more variable to get today's date.